DataFormats/Detectors/ZDC/include/DataFormatsZDC/RecEventFlat.h

```diff
@@ -99,6 +100,32 @@ struct RecEventFlat { // NOLINT: false positive in clang-tidy !!
   void init(const gsl::span<const o2::zdc::BCRecData> RecBC, const gsl::span<const o2::zdc::ZDCEnergy> Energy, const gsl::span<const o2::zdc::ZDCTDCData> TDCData, const gsl::span<const uint16_t> Info);
 
   int next();
+  int at(int ientry);
+
+  void allocate(int isig)
+  {
+    if (inter[isig].size() != NIS) {
+      inter[isig].resize(NIS);
+      for (int iis = 0; iis < NIS; iis++) {
+        inter[isig][iis] = 0;
+      }
+    }
+  }
```

Comment:
Thanks for the suggestion about initialization.
I use vectors because in normal runs the inter[isig] data is not filled and therefore I prefer not to allocate space for them. They are quite large arrays with 2400 elements each.

Reply:
ok, then just remove 0 assignments.